dialogue-apache-hc5-client/src/test/java/com/palantir/dialogue/hc5/ApacheHttpClientChannelsTest.java

```diff
@@ -143,16 +145,50 @@ public void countsUnknownHostExceptions() throws Exception {
             ListenableFuture<Response> future =
                     channel.execute(TestEndpoint.GET, Request.builder().build());
 
-            try (Response response = Futures.getChecked(future, UnknownHostException.class)) {
+            try (Response response = Futures.getUnchecked(future)) {
                 fail("This request should have failed with an unknown host exception! (code: %d)", response.code());
-            } catch (UnknownHostException _exception) {
-                // No-op, this is expected
+            } catch (UncheckedExecutionException exception) {
+                assertThat(exception.getCause()).isInstanceOf(UnknownHostException.class);
```

Comment:
Was a silly mistake of mine; `getChecked` wraps the exception (`converting most exceptions to a new instance of the given checked exception type`). This ensures it's of the expected type.